### PR TITLE
Sdp getters2

### PR DESCRIPTION
**Pull request by trigaut**
This PR adds get_control in HD and DH case as well as tests and minor fixes

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -434,15 +434,47 @@ facts("SDP algorithm") do
...
         costs_sdp, stocks_sdp, controls_sdp = sdp_forward_simulation(modelSDP,
                                                                 paramsSDP,
                                                                 aleas_scen, x0,
                                                                 V_sdp, true )
 
+        @fact (get_control(modelSDP,paramsSDP,V_sdp3, 1, x)[1] >= CONTROL_MIN) --> true
+        @fact (get_control(modelSDP,paramsSDP,V_sdp3, 1, x)[1] >= CONTROL_MIN) --> true
+
         @fact size(stocks_sdp) --> (3,1,2)
         @fact size(controls_sdp) --> (2,1,2)
```
Comment (Henri-Gerard) [inline test/runtests.jl:480]: It's ok for me. If the others have no remarks, we can merge this PR

File: examples/stock-example.jl [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+paramSDP = SDPparameters(spmodel, stateSteps, controlSteps, infoStruct)
+Vs = sdp_optimize(spmodel,paramSDP)
+lb_sdp = StochDynamicProgramming.get_value(spmodel,paramSDP,Vs)
+println("Lower bound obtained by SDP: "*string(lb_sdp))
+
+######### Comparing the solution
+#scenarios = StochDynamicProgramming.generate_scenarios(xi_laws,1)
+#costs, stocks = forward_simulations(spmodel, params, V, pbs, scenarios)
+
```
Comment (leclere) [inline examples/stock-example.jl]: I don't know what you changed in this file. Maybe don't put it in the PR.

Comment (trigaut) [inline examples/stock-example.jl]: I removed it

File: src/SDPoptimize.jl [hunk 3]
```diff
@@ -117,8 +117,51 @@ function generate_grid(model::SPModel, param::SDPparameters)
...
+
+- param (SDPparameters)
+    the parameters of the problem
+
+
+Returns :
+- sdpmodel : (StochDynProgModel)
+    the corresponding StochDynProgModel
+"""
+function SPmodel_to_SDPmodel(model::SPModel, param::SDPparameters)
```
Comment (leclere) [inline src/SDPoptimize.jl:134]: I dont think param is required.

File: src/SDPoptimize.jl [hunk 4]
```diff
@@ -369,30 +454,155 @@ Parameters:
...
+- t (int)
+    the time step
+
+- x (Array)
+    the state variable
+
+Returns :
+- V(x0) (Float64)
+"""
+function get_control(model::SPModel,param::SDPparameters,V::Array{Float64}, t::Int64, x::Array)
```
Comment (leclere) [inline src/SDPoptimize.jl:491]: I suggest that this function is only for SDPmodel.

File: src/SDDPoptimize.jl [hunk 5]
```diff
@@ -123,9 +123,8 @@ function run_SDDP(model::SPModel,
 
         if (display > 0) && (iteration_count%display==0)
             println("Pass number ", iteration_count,
-                    "\tEstimation of upper-bound: ", upper_bound(costs),
-                    "\tLower-bound: ", get_bellman_value(model, param, 1, V[1], model.initialState),
-                    "\tTime: ", toq())
+                    "\tLower-bound: ", round(get_bellman_value(model, param, 1, V[1], model.initialState),4),
```
Comment (frapac) [inline src/SDDPoptimize.jl:126]: Did you branch from stock-example?

File: src/SDPoptimize.jl [hunk 6]
```diff
@@ -320,7 +407,7 @@ function sdp_solve_HD(model::SPModel,
             #Compute expectation
             for w in 1:sampling_size
                 admissible_u_w_count = 0
-                best_V_x_w = 0.
+                best_V_x_w = Inf
```
Comment (frapac) [inline src/SDPoptimize.jl:410]: :)

File: src/SDDPoptimize.jl (hunk 5 above)
Comment (trigaut) [inline src/SDDPoptimize.jl:126]: Yes

File: src/SDPoptimize.jl (hunk 6 above)
Comment (trigaut) [inline src/SDPoptimize.jl:410]: My best piece of work

**Pull request closed without merging**